mne/export/_edf.py

```diff
@@ -60,7 +29,6 @@ def _export_raw(fname, raw, physical_range, add_ch_type):
 
     digital_min = -32767
```

Comment:
Yes, that would be consistent with the EDF specs and maximize the resolution. I wasn't sure if this is an intended deviation from the specs as it allows to trivially set physical min and max so that `0` stays `0` after export and re-import :thinking:

Reply:
I don't know if this was intentional, but if you think that it is a good idea to keep the symmetry, losing one value of resolution is not a big deal.

Reply:
Yes I would keep this as-is, to avoid breaking workflows relying on that behavior.